packages/sol-compiler/src/compiler.ts

```diff
@@ -1,26 +1,19 @@
 import { assert } from '@0xproject/assert';
-import {
-    FallthroughResolver,
-    FSResolver,
-    NameResolver,
-    NPMResolver,
-    RelativeFSResolver,
-    Resolver,
-    URLResolver,
-} from '@0xproject/sol-resolver';
-import { fetchAsync, logUtils } from '@0xproject/utils';
+import { NameResolver, Resolver } from '@0xproject/sol-resolver';
+import { logUtils } from '@0xproject/utils';
 import chalk from 'chalk';
+import * as child_process from 'child_process';
```

Comment:
Let's import it as `childProcess` since that suites our naming conventions and is inconsequential given that we are already importing it using `* as`.